react-function-components/src/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const button = <button>Click Me!</button>;
```

Comment:
Please define a function component for this exercise.

Reply:
Issue resolved with new commit f3b6917